### PR TITLE
chore(readme): manual installation; typo

### PR DESCRIPTION
**Pull request by greyscaled**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,9 @@ more without moving data around.
 curl https://glaredb.com/install.sh | sh
 ```
 
-Or check out [console.glaredb.com](https://console.glaredb.com) for full managed deployments of GlareDB.
+Or check out [console.glaredb.com](https://console.glaredb.com) for fully managed deployments of GlareDB.
```
Comment (greyscaled) [inline README.md:31]: typo: `full` -> `fully`

File: README.md [hunk 2]
```diff
@@ -28,7 +28,9 @@ more without moving data around.
...
+If you prefer manual installation, download, extract and run the GlareDB binary from a release in our
+[releases page](https://github.com/GlareDB/glaredb/releases).
```
Comment (greyscaled) [inline README.md:33]: To maintain link to releases as optional.

LMK if prefer to just can it entirely.

**Review by scsmithr — APPROVED** [review]
take away my push perms

**Comment by greyscaled** [comment]
> take away my push perms

never

**Pull request merged**